src/api/public/apidocs-new/paths/my_notifications_id.yaml

```
@@ -0,0 +1,31 @@
+put:
+  summary: Toggle the read/unread state of a notification.
+  description: |
+    Toggle the boolean value of the `delivered` attribute of a notification, which
+    marks the notification as read/unread.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: path
+      name: id
+      required: true
+      schema:
+        type: int
```

This should be `integer`, not `int`. See https://swagger.io/docs/specification/data-models/data-types/

```suggestion
        type: integer
```